### ISSUE TITLE
post-disconnect.sh stdout/stderr not captured to logs

### ISSUE THREAD

**Issue by jankcat** [Tunnelblick/Tunnelblick#344]
post-disconnect.sh's stdout/stderr are not captured to the TunnelBlick logs, or at least not output to the Log tab in the UI. 

All that is output for the script is:
``` 
2016-12-06 11:54:11 *Tunnelblick: The 'post-disconnect.sh' script executed successfully
```

I am no objective-c expert, but would something like the following change in the "runScriptNamed" function do the trick?

At line [3497](https://github.com/Tunnelblick/Tunnelblick/blob/master/tunnelblick/VPNConnection.m#L3497) of tunnelblick/VPNConnection.m, change this:
```
[self addToLog: [NSString stringWithFormat: @"*Tunnelblick: The '%@.sh' script executed successfully", scriptName]];
```
to this:
```
[self addToLog: [NSString stringWithFormat: @"*Tunnelblick: The '%@.sh' script executed successfully", scriptName]];
NSString * runScriptNamedLogOutput;
runScriptNamedLogOutput = stringForLog(stdErrString, [NSString stringWithFormat: @"*Tunnelblick: '%@.sh' runOpenvpnstart log:\n", scriptName]);
if (  runScriptNamedLogOutput  ) {
	if (  [runScriptNamedLogOutput length] != 0  ) {
		[self addToLog: runScriptNamedLogOutput];
	}
}   
```

Once again, I am no Obj-C expert, nor am I super familiar with the TunnelBlick code, so take it with a grain of salt...

**Comment by jkbullard** [comment]
Thanks for the suggestion. You are correct:
 * If the script succeeds (returns a status of zero), then the stdout and stderr outputs from the script are discarded.
 * If the script fails (returns a non-zero status)Tunnelblick adds both stdout and stderr outputs from the script to the log.

I agree that it probably makes sense to include stdout and stderr output from scripts whether or not the script has failed. (Of course, stderr will presumably be empty if the script has succeeded.)

I'll add it to my "to do" list.

**Comment by jkbullard** [comment]
This was included in 0643ce14cc191b636542dfc5bd01d055fa767042 and was in Tunnelblick 3.6.10beta05 build 4740 (2016-12-22).

**Comment by jankcat** [comment]
<3 Awesome, thanks!
